src/tooltip/tooltip.js

```diff
@@ -443,7 +490,14 @@ angular.module('ui.bootstrap.tooltip', ['ui.bootstrap.position', 'ui.bootstrap.s
                 triggers.show.forEach(function(trigger, idx) {
                   // Using raw addEventListener due to jqLite/jQuery bug - #4060
                   if (trigger === triggers.hide[idx]) {
-                    element[0].addEventListener(trigger, toggleTooltipBind);
+                    if (trigger === 'clickaway') {
```

Comment:
I don't think this should be nested, this would read clearer as an additional else if?